waspc/data/Generator/templates/sdk/wasp/server/email/core/providers/sendgrid.ts

```diff
@@ -4,7 +4,7 @@ import type { SendGridProvider, EmailSender } from "../types";
 
 // PRIVATE API
 export function initSendGridEmailSender(
-  provider: SendGridProvider
+    provider: SendGridProvider
```

Comment:
Let's remove this indent to keep the changeset minimal.

Reply:
Whoops, formatters at work I guess.